README.md

```diff
@@ -38,31 +39,27 @@ So it must be installed on the system to compile and use those crates.
 
 Using the [`proj` crate] requires some system-dependencies installation.
 
-* The version `6.3.0` of [PROJ] is needed (used and tested by maintainers).<br>
-  On Debian systems:
-  ```sh
-  # Needed only for proj install
-  sudo apt install -yq wget build-essential pkg-config sqlite3 libsqlite3-dev
-
-  sudo apt remove libproj-dev  # remove system version from packages
-  wget https://github.com/OSGeo/proj.4/releases/download/6.3.0/proj-6.3.0.tar.gz
-  tar -xzf proj-6.3.0.tar.gz
-  cd proj-6.3.0
-  ./configure --prefix=/usr
-  make
-  sudo make install
-
-  # These dependencies can be removed after PROJ build
-  # sudo apt purge wget build-essential pkg-config sqlite3 libsqlite3-dev
-  ```
-  [PROJ installation instructions](https://github.com/OSGeo/PROJ#installation)
-  may help, too.
-
-* Some packages are also needed.<br>
-  On Debian systems:
-  ```sh
-  apt install -y clang libssl-dev
-  ```
+The version `6.3.0` of [PROJ] is needed (used and tested by maintainers).<br>
+On Debian systems:
+
+```sh
+# Needed only for proj install
+sudo apt install -yq wget build-essential pkg-config sqlite3 libsqlite3-dev
+
+sudo apt remove libproj-dev  # remove system version from packages
+wget https://github.com/OSGeo/proj.4/releases/download/6.3.0/proj-6.3.0.tar.gz
+tar -xzf proj-6.3.0.tar.gz
+cd proj-6.3.0
+./configure --prefix=/usr
+make
+sudo make install
+
+# These dependencies can be removed after PROJ build
+# sudo apt purge wget build-essential pkg-config sqlite3 libsqlite3-dev
```

Comment:
If cleanup is done, clang is still needed for compile (libssl-dev doesn't seem required)
```suggestion
# sudo apt purge wget build-essential pkg-config sqlite3 libsqlite3-dev && sudo apt install -yq clang
```

Reply:
I suggest to remove this line about cleanup